src/components/font_size_editor/font_size_editor.xml

```diff
@@ -1,24 +1,31 @@
 <templates>
   <t t-name="o-spreadsheet-FontSizeEditor" owl="1">
-    <div class="o-dropdown o-font-size-editor" t-ref="FontSizeEditor">
-      <input
-        type="number"
-        min="1"
-        max="400"
-        class="o-font-size bg-transparent border-0 ps-1"
-        t-on-keydown="onInputKeydown"
-        t-on-click.stop=""
-        t-on-focus.stop="onInputFocused"
-        t-att-value="currentFontSize"
-        t-on-change="setSizeFromInput"
-        t-ref="inputFontSize"
-      />
-      <div title="Font Size" t-on-click="() => this.toggleFontList()">
-        <t t-call="o-spreadsheet-Icon.TRIANGLE_DOWN"/>
+    <div class="o-dropdown" t-ref="FontSizeEditor">
+      <div
+        class=" o-font-size-editor h-100 d-flex align-items-center"
```

Comment:
why do we need `d-flex` and `align-items-center` here and not in the saas-16.2 ? These property has been removed/replaced with bootstrap use between the two versions ?

Reply:
This is included in the `o-tool ` class that we give to the componenet in 16.2, class that we don't give in 16.3